testkit/src/main/scala/com/raphtory/BaseRaphtoryAlgoTest.scala

```diff
@@ -23,9 +28,29 @@ abstract class BaseRaphtoryAlgoTest[T: ClassTag: TypeTag](deleteResultAfterFinis
   protected val logger: Logger = Logger(LoggerFactory.getLogger(this.getClass))
 
   var jobId: String           = ""
-  val outputDirectory: String = "/tmp/raphtoryTest"
-  def defaultSink: Sink       = FileSink(outputDirectory)
+  val outputDirectory: String = Option(System.getenv("IT_OUTPUT_PATH")).getOrElse("/tmp/raphtoryTest")
```

Comment:
Think all of our Env vars start with RAPHTORY_ may be worth keeping the trend for this.

Reply:
fair